pdcupdater/services.py

```diff
@@ -72,13 +75,19 @@ def old_composes(base_url):
 
 
 @pdcupdater.utils.with_ridiculous_timeout
-def fas_persons(base_url, username, password):
+def fas_persons(base_url, username, password, fasjson):
     """ Return the list of users in the Fedora Account System. """
 
+    log.info("Connecting to Account System at %r" % base_url)
+    if fasjson:
+        client = Client(url=base_url)
+        response = client.list_users().result
+        people = response.get('result', [])
```

Comment:
Hmm no you don't need that I think. The data is directly available in the `.result` property.